extensions/sql-bindings/src/extension.ts

```diff
@@ -17,7 +18,19 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 	context.subscriptions.push(vscode.commands.registerCommand('sqlBindings.addSqlBinding', async (uri: vscode.Uri | undefined) => { return launchAddSqlBindingQuickpick(uri); }));
 	// Generate Azure Function command
 	context.subscriptions.push(vscode.commands.registerCommand('sqlBindings.createAzureFunction', async (node: ITreeNodeInfo) => {
-		let connectionInfo = node.connectionInfo;
+		if (!node) {
+			let openServerPane = await vscode.window.showErrorMessage(constants.chooseAConnection,
+				constants.openPane, constants.learnMore);
+			if (openServerPane === constants.openPane) {
+				// open SQL Server connections pane
+				await vscode.commands.executeCommand('objectExplorer.focus');
```

Comment:
I am a bit confused about what you mean here? Using the promptForConnection to list connections and then connect to the selected connection and open the objectExplorer for the user to see?

In VS Code this would open up the pane.

Reply:
Yes, using promptForConnection is what I meant (and then continuing on with the connection info that it returns back)

I though this was for ADS at first - so then my question is "what about ADS"? The plan is to publish this there as well, right?

Reply:
Not sure what you mean by continuing with the connection info. We would need the schema and table to pass into the `createAzureFunction` below. That is why I stated in the description it would be a future feature where we would have to prompt for databases (using API) and then tables/views (which have not been created).

The plan is to publish there as well, however, all the use cases as far as I know are for VS Code.

Reply:
Gotcha, missed/forgot about the part where you need to prompt for the table/view. We could just have them type it in manually too - we already do this in other places. And given that we still need to prompt for a connection regardless that seems like a good thing to get in now if we can.

What do you mean the "use cases"? The commands can be ran in both VS Code and ADS so should function in both (or be disabled if they aren't applicable). Have you tested this in ADS at all?

Reply:
I'll go ahead and do that now. 

In terms of use cases, we targeted VS code development. Would there be a way to remove the command from ADS specifically?

Reply:
`"when": "!azdataAvailable"`

Reply:
When are you planning on adding ADS support? That shouldn't be too hard to do and it would be good to keep the feature set at parity when possible.

Reply:
I want to focus on vscode support first. Once all these PRs are in I can see if I can put this as a priority.